worlds/alttp/__init__.py

```diff
@@ -827,11 +833,21 @@ def fill_slot_data(self):
                             "triforce_pieces_available", "triforce_pieces_extra",
             ]
 
-            slot_data = {option_name: getattr(self.multiworld, option_name)[self.player].value for option_name in slot_options}
+            for dungeon in self.dungeons.keys():
+                if dungeon not in ['Hyrule Castle', 'Ganons Tower', "Agahnims Tower"]:
+                    bosses[dungeon] = self.dungeons[dungeon].boss.name
+
+                    if dungeon == "Thieves Town":
+                        dungeon = "Thieves\' Town"
+                    prizes[dungeon] = self.get_location(location_name=dungeon+" - Prize").item.name
```

Comment:
This and one spot in OoT should use f-strings and not concatenation
```suggestion
                    prizes[dungeon] = self.get_location(f"{dungeon} - Prize").item.name
```